fe/fe-core/src/main/java/com/starrocks/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -220,18 +219,6 @@ protected boolean checkCommitInfo(RLTaskTxnCommitAttachment rlTaskTxnCommitAttac
             return true;
         }
 
-        if (txnStatusChangeReason != null &&
-                txnStatusChangeReason == TransactionState.TxnStatusChangeReason.NO_PARTITIONS) {
```

Comment:
Should not remove because the default behavior is still abort when no data loaded.
Add some comments for this.

Reply:
The default behaviour of empty transaction in routine load should be changed to commit successfully. Otherwise, #2862 cannot be resolved.